Add option to include helpful code comments

tBoccinfuso opened the pull request:
The page now includes a checkbox that will auto-generate very useful code comments for your snippet. 

Each case in the switch will require the new helper function found on the Comments class. This is normally added right before the fillerLine loops. I had added it to each case at the time of this pull request.

The `generateRandomCode` function now takes in a 3rd param called `addComment` this is the state we pass from the page.

Example below:
```javascript
      if (addComment) {
        fillerLines.push(Comments.getRandomComment());
      }
```


Note: Probably can use some styling improvements for the checkbox.

whitep4nth3r commented:
#### 📝Changed routes:


| [**/**](https://randomcodegenerator-lol-dohtozkvc-whitep4nth3r.vercel.app/) |
|:-:|
|<a href="https://randomcodegenerator-lol-dohtozkvc-whitep4nth3r.vercel.app/"><img src="https://images.weserv.nl?url=https://s.microlink.io/gS07_x6cY16kTvWD8AguHHk1DETiPi7qvI6cnjsgqcMzM-rZJSx2DGssqMd7ucFUSh7eYmnfQN--jTBb_iLOcg.png&w=600" alt="Screenshot of /" width="300"></a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/D7Erk1dK0Cji3NbHBAhqEiddDy7zSEIGFOJwrCyqNye74JK9V3O9HWIhpL9zvnAx_4tbVzYMJ3kg0aWVBmJiYA.png">(view full size)</a> |







Commit 5f039ad675e1661c7b9f4ce49986bce1ab979e4f (https://randomcodegenerator-lol-dohtozkvc-whitep4nth3r.vercel.app).

whitep4nth3r commented:
I think it might be more fun to just have the comment functionality as one of the random lines that might be generated. GOOD WORK, FRIEND!

tBoccinfuso commented:
> I think it might be more fun to just have the comment functionality as one of the random lines that might be generated. GOOD WORK, FRIEND!

Added logic in so a random comment will be generated! Also updated the `getRandomComment` function to allow you to pass in a string (languageOverride). We can use this value in a new switch statement to adjust comments so they will be valid for each language.

whitep4nth3r commented:
Hello! Thanks for the updated PR! Please rebase with main and use the new helpers.js file to generate your random things!

whitep4nth3r commented on tools/RandomCodeGenerator.js:
```diff
@@ -69,6 +75,11 @@ export function generateRandomCode(language, lines) {
 
       fillerLines.push("{");
+
+      if (addComment) {
```
Should we alter the fillerLineQty if the addComment is true? Otherwise, we'll be getting the wrong number of lines.

whitep4nth3r commented:
This is brilliant! I've added one more comment about the fillerLineQty. (The logic needed to alter the qty in JS will be different as currently JS is the only lang with 'return var' support - which alters fillerLineQty.

whitep4nth3r commented:
Hello! Please could you rebase with main and fix up some conflicts? I just merged in a PR that reorganised some of the constants and things, which has made contributing easier in the long run. Thanks for your PR!

whitep4nth3r commented:
@all-contributors please add @tBoccinfuso for code
